marklogic-data-hub-central/ui/src/components/queries/managing/manage-query-modal/manage-query.test.tsx

```diff
@@ -38,7 +39,14 @@ describe('Query Modal Component', () => {
   });
 
   test('Verify link column does not exist', () => {
-    const { queryByTitle } = render(<ManageQuery {...defaultProps} modalVisibility={true} />);
-    expect(queryByTitle('Link')).toBeNull();
+    const { queryByTestId } = render(<ManageQuery {...defaultProps} isSavedQueryUser={true} />);
+    expect(queryByTestId('link')).toBeNull();
+  });
+
+  test('Verify confirmation modal message for deleting a query', () => {
+    const { getByTestId, getByText } = render(<ManageQuery {...defaultProps} isSavedQueryUser={true} />);
+    expect(getByTestId('delete')).toBeInTheDocument();
+    fireEvent.click(getByTestId('delete'));
+    expect(getByTestId('deleteConfirmationText')).toBeInTheDocument();
```

Comment:
We would want to test just like a user would, by looking at the delete confirmation text and not the implementation detail, to align with RTL best practices.
You can try `getByText((_, node) => node.textContent == 'your text here')` to check if it asserts text with bold content. Otherwise refer to `getSubElements` function in Run.test.tsx that asserts text in child nodes as well.